finder/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "dependencies": {
+    "bulma": "^0.7.1",
+    "jquery": "^3.3.1",
```

Comment:
We should get rid of jquery but after this PR is merged.

Reply:
need to find a way of doing get requests without much hazzle 👍

Reply:
maybe using fetch https://developers.google.com/web/updates/2015/03/introduction-to-fetch